### DISCUSSION TITLE
"Remember me" option when logging in

### DISCUSSION THREAD

**Discussion by Neriderc** [goauthentik/authentik#2761] [Ideas]
I had thought I'd seen this talked about, but not can't find any issue or discussion that asks for it: Can we get a "remember me" checkbox when logging in so that we don't have to continuously log in on the same device?

**Comment by unixfy** [comment]
As a sort of workaround you can configure the session length on your user login stage.

![image](https://user-images.githubusercontent.com/12488860/172910192-b306311c-410a-4d4c-90a3-8d60fcd5c514.png)

**Comment by linkandzelda91** [comment]
this may not have been a feature in april 2022, but it sure is now, on the login stage:
![image](https://github.com/user-attachments/assets/31cc72e8-c9fb-4457-a61f-9f851ede7bf0)
